app/views/problems/_sparkline.html.haml

```diff
@@ -0,0 +1,9 @@
+= now_as_seconds_since_epoch = Time.current.to_i
+= now_as_seconds_since_epoch = 8.weeks.ago.to_i
```

Comment:
Duplicate definition of variable - think this line 2 needs to go?

Reply:
resolved, thanks for catching this!